content/blog/posts/for-the-love-of-the-game.md

```diff
@@ -0,0 +1,109 @@
+
+---
+title: "For the Love of the Game: In Search of Best-of-Breed 3D Creation Tools"
+date: 2022-09-16
+slug: for-the-love-of-the-game
+author: Nicole Huesman
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+blog_img: "/images/blog/announcement_thumbnail.jpg"
```

Comment:
I would suggest having a Blog image instead of defaulting to the bland default image. This image is very outdated and needs a refresh.